fe/fe-core/src/main/java/org/apache/doris/nereids/cost/CostCalculator.java

```diff
@@ -72,13 +72,12 @@ public CostEstimate visitPhysicalOlapScan(PhysicalOlapScan physicalOlapScan, Pla
         }
 
         @Override
-        public CostEstimate visitPhysicalProject(PhysicalProject physicalProject, PlanContext context) {
-            StatsDeriveResult statistics = context.getStatisticsWithCheck();
-            return CostEstimate.ofCpu(statistics.computeSize());
+        public CostEstimate visitPhysicalProject(PhysicalProject<Plan> physicalProject, PlanContext context) {
+            return CostEstimate.zero();
```

Comment:
i rethink here, the better way is return a light cost than zero, to let us choose the plan include less project when we intro merge project rule into optimize stage